### ISSUE TITLE
Do we need to do dereverberation?

### ISSUE THREAD

**Issue by ujscjj** [microsoft/DNS-Challenge#8]
Hello, thank you for this challenge.
In the 'datasets/test set/synthetic/with reverb/' folder,  there is dereverberation in the clean speech.
So, I think that the aim of this challenge is to remove the noise, but not including removing the reverberation. Right?

**Comment by ujscjj** [comment]
On the other hand, it seems that you didnot add reverberation in the training set. In my opiniom, this will cause a mismatch between the training and test sets.

**Comment by chandanka90** [comment]
Doing dereverb will definitely improve quality. Every real recording will have some level of reverberation. Even the speech and noise in the training sets will have some reverb as they were not collected in the anechoic chamber. But you are more than welcome to add more reverb to the training set. I would recommend you to add reverb only to clean speech and not to noise. Also, make sure to cover a wide range of RT60 with probably a sub-Gaussian distribution with mean of 300 ms.

**Comment by jkzhang7** [comment]
@chandanka90 Very nice work! I wonder how to determine Impulse Response(IR) parameter if we would like to add reverb effect in the training dataset. Thanks a lot!

**Comment by chandanka90** [comment]
Check this work https://ieeexplore.ieee.org/abstract/document/9287158
